src/DependencyInjection/Configuration.php

```diff
@@ -531,12 +531,9 @@ private function getHttpDispatcherNode()
         $node
             ->fixXmlConfig('server')
             ->children()
-                ->arrayNode('servers')
+                ->variableNode('servers')
                     ->info('Addresses of the hosts the caching proxy is running on. May be hostname or ip, and with :port if not the default port 80.')
-                    ->useAttributeAsKey('name')
```

Comment:
afaik this is breaks xml configuration.

i wonder if it would be best to make this node not required and add a separate node for the env variable and have the Configuration class check that exactly one of the two nodes is set. and in the extension pick the one we have.

that would not mess with xml or the array validation.